otx/algorithms/common/adapters/mmcv/ops/multi_scale_deformable_attn_pytorch.py

```diff
@@ -0,0 +1,136 @@
+"""Custom patch of multi_scale_deformable_attn_pytorch for openvino export."""
+
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+import torch
+import torch.nn.functional as F
+
+
+def multi_scale_deformable_attn_pytorch(
+    value: torch.Tensor,
+    value_spatial_shapes: torch.Tensor,
+    sampling_locations: torch.Tensor,
+    attention_weights: torch.Tensor,
+) -> torch.Tensor:
+    """Custom patch for multi_scale_deformable_attn_pytorch function.
+
+    Original implementation in mmcv.ops use torch.nn.functional.grid_sample.
+    It occurs errors during inference with OpenVINO exported model.
+    Therefore this function change grid_sample function to custom mmcv_grid_sample function.
```

Comment:
Is this function same except using `_custom_grid_sample`?
If then, I think writing that information in comments can makes code maintenance easier later.

Reply:
Changed from custom mmcv_grid_sample -> _custom_grid_sample